### DISCUSSION TITLE
How to overwrite default style property conditionally?

### DISCUSSION THREAD

**Discussion by anki247** [chakra-ui/chakra-ui#2745] [Q&A]
Given I just want to change the border-color of `<Input>` only if some condition matches. Else it should keep the color from theme (respecting color mode)
I tried this, but it does not work...  `<Input borderColor={isHighlighted ? 'teal.500' : undefined} />`
Is there any solution for this?
I also would like to change the color of `_hover`

Thx in advance

**Comment by yudyananda** [comment]
@anki247  Did you find solution?

**Reply by anki247** [reply]
I didn't try this out but this might be helpful... 
https://twitter.com/pagebakers/status/1555216754082971648?s=20&t=RaKJuwBGHJoRrZsQLYT-8A

**Comment by anubra266** [comment]
use `focusBorderColor` prop

**Comment by VentusTWY** [comment]
try returning empty object {} instead of undefined
